src/middleware/userAgentFilter.ts

```
@@ -0,0 +1,68 @@
+  /sqlmap/,
+  /dirbuster/,
+  /gobuster/,
+  /ffuf/,
+  /wfuzz/,
+  /hydra/,
+  /medusa/,
+  // Headless / scripted abuse tools
+  /python-requests/,
+  /go-http-client/,
+  /libwww-perl/,
+  /curl\/[0-9]/,   // bare curl (not curl wrapped in a real UA)
```

ESLint

[error] Delete `··`

(prettier/prettier)